### DISCUSSION TITLE
Ignoring whitespace in summary window

### DISCUSSION THREAD

**Discussion by marlun** [tpope/vim-fugitive#2139] [Q&A]
When removing a parent element in HTML which changes the indentation for a whole section, git shows the whole section as chaning. However if I add -w it shows a diff which is much easier to understand in this case. When Doing `:G` and showing the status window, moving to such file and hit `=` to show the diff, can I somehow make it ignore whitespace? I tried adding `iwhiteall` to `diffopt` but that did not seem to help.

**Comment by tpope** [comment]
Not possible, because those diffs are load bearing: The ability to partially stage a change hinges on the ability to patch part of a diff to `git apply`, and a diff that's ignoring whitespace doesn't contain enough information to serve as a patch.
